docs/README.building

```diff
@@ -0,0 +1,119 @@
+FreeRDP has a few dependencies that are required for proper operation:
+
+1. SSL
+
+RDP requires a secure tunnel and utilizes TLS for this. We do not implement this
+ourselves but reuse existing libraries:
+
+We support
+
+* OpenSSL our main development SSL library
+* LibreSSL (supported by community)
+* MBedTLS (supported by community)
+
+2. Kerberos
+
+Authentication to many services requires kerbers (especially if smartcards are in use)
+
+We support:
+
+* MIT
+* Heimdal
+
+3. JSON
+
+Azure logon requires HTTP/JSON messages to be parsed.
+
+We support:
+
+* cJSON
+
+4. H264
+
+RDP GFX modes (anything newer Windows 8.1 / Server 2008) supports a graphics mode based
+on the H264 codec
+
+We support
+
+* OpenH264
+* FFMPEG (x264 or OpenH264)
+
+There are some platform specific implementations too (e.g. mediacodec on android) but these
+two are the options that are always required.
+
+5. Graphics scaling support
+
+High DPI modes and the client side smart-sizing option require bitmaps to be scaled. 
```

Comment:
Let's add one more item here: "and the ability to let the server side to scale content". The 105, 106, and 107 GFX caps require the client side to be able to scale bitmaps.

Reply:
rephrased.